OWNERS

```diff
@@ -4,3 +4,4 @@ approvers:
 reviewers:
   - bparees
   - kyoto
+  - sauagarwa
```

Comment:
being listed in reviewers doesn't actually grant any permissions, in fact this is what i put in our other owners files:

https://github.com/openshift/lightspeed-service/blob/07fd9ea3ab3f591a7c077ef3ee1c98b73dc28ca7/OWNERS#L6

all it accomplishes is that it hints to prow who to assign the PR to for review, which i find to be arbitrary and unhelpful.

if @sauagarwa is not a member of the openshift github org, then he should request to be added via a ticket here:
https://devservices.dpp.openshift.com/support/  (site currently seems to be down), there's an option for requesting to be added to a github org.

any member of the openshift github org can "/lgtm" a PR.

if you want @sauagarwa to be able to "/approve" a PR, then he should be listed as an approver.

here is my guidance on approving vs lgtm'ing PRs:

https://docs.google.com/document/d/1TGRfeAGZK3OabYORAGU4m2WVWZiE2i582k2V7sKS9oY/edit#heading=h.m5pxiaws9ajy